test/epoll1.c

```diff
@@ -54,7 +54,7 @@ void server()
       exit(EXIT_FAILURE);
     }
 
-    socklen_t addrlen;
+    socklen_t addrlen = sizeof(struct sockaddr);
```

Comment:
Ideally, it should be `sizeof(sockaddr)` since `struct sockaddr` and `struct sockaddr_in` could be of different sizes but I think it's fine here for this test.